plane/source/docs/guide-tilt-rotor.rst

```diff
@@ -0,0 +1,231 @@
+specific documentation.
+
+In ArduPilot nomenclature, a tilt-rotor is a type of VTOL aircraft
+where transition between hover and forward flight is accomplished by
+tilting one or more rotors so that it provides forward thrust instead
+of upward thrust.
+
+This is distinct from :ref:`tailsitters <guide-tailsitter` where the
+autopilot and main fuselage change orientation when moving between
+hover and forward flight.
+
+Types of Tilt-Rotors
+====================
+
+ArduPilot supports a very wide range of tilt-rotor
+configurations. Common configurations include:
+
+- tilt-quadplanes with the front two motors tilting
+- tilt-quadplanes with all four motors tilting
+- tilt-tricopters with the front two motors tilting and rear tilt for
+  yaw
+- tilt-tricopters with the front two motors tilting and vectored yaw
+- tilt-hexacopters with the front four motors tilting
+- tilt-wings where the main wing tilts along with two motors
+- binary-tiltrotors where the tilt mechanism can only be in one of two positions
+- continuous-tiltrotors where the tilt mechanism can be controlled to
+  any angle in a range from straight up to straight forward
+- vectored tilt-rotors where the tilt of the rotors on the left can be
+  controlled independently from the tilt of the right motors
+
+Combined with these varients are versions that use ailerons, elevons,
+vtails and other control surfaces for fixed wing flight. There are an
+amazing number of combinations possible, and experimentation with VTOL
+designs is common. ArduPilot aims to support a very wide range of
+tilt-rotor configurations with a small number of parameters.
+
+Setting Up A Tilt-Rotor
+=======================
+
+The first thing you need to do is enable QuadPlane support by setting
+Q_ENABLE to 1, and then choose the right quadplane frame class and
+frame type.
+
+The quadplane frame class is in Q_FRAME_CLASS. The frame class is
+chosen based on your vehicles rotor configuration while
+hovering. Currently supported tilt-rotor frame classes are:
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tr><th>Frame Class</th><th>Q_FRAME_CLASS</th></th></tr>
+   <tr><td>Quadcopter</td><td>1</td></tr>
+   <tr><td>Hexacopter</td><td>2</td></tr>
+   <tr><td>Octacopter</td><td>3</td></tr>
+   <tr><td>Octaquad</td><td>4</td></tr>
+   <tr><td>Y6</td><td>5</td></tr>
+   <tr><td>Tricopter</td><td>7</td></tr>
+   </table>
+
+Once you have chosen your frame class you will need to get the
+Q_FRAME_TYPE right. The Q_FRAME_TYPE is the sub-type of frame. For
+example, for a quadcopter, a frame type of 1 is for a "X" frame and a
+frame type of 3 is for a "H" frame.
+
+Please see the ArduCopter setup guide for multi-copters for more
+information on choosing your frame type.
+
+After setting up Q_ENABLE, Q_FRAME_CLASS and Q_FRAME_TYPE you will
+need to reboot.
+
+The Tilt Mask
+=============
+
+The most important parameter for a tilt-rotor is the tilt-mask, in the
+Q_TILT_MASK parameter.
```

Comment:
This needs a link to the params page. Also, the description for this param could use some expanding or at least linking to the standard motor order page.